src/content/docs/synthetics/synthetic-monitoring/getting-started/new-runtime.mdx

```diff
@@ -0,0 +1,100 @@
+* The ability for your monitors to treat objects like frames and find objects inside the shadow DOM.
+* [Custom timing library](SOMETHING) for scripted API monitors.
+- Access to automated management of your synthetic monitors using [NerdGraph APIs](SOMETHING). 
+
+We recommend converting your legacy runtime to take advantage of these capabilities. Eventually, we plan to make this new runtime available for private monitoring locations. 
+
+<Callout variant="important">
+Before switching, make sure that you've updated your allow lists to accept network requests from the new runtime’s [public IP ranges](/docs/synthetics/synthetic-monitoring/administration/new-synthetic-public-minion-ips).
+</Callout>
+
+## Convert your monitors to the new runtime [#runtime-convert]
+
+To prevent your critical monitors from breaking during future end of life processes, we recommend you convert your public monitors sooner rather than later. To convert:
+
+1. Go to [one.newrelic.com](https://one.newrelic.com/nr1-core), click **Synthetics**, then select the monitor you want to edit.
+2. From the **Settings** tab, click **General**. 
+3. Use the dropdown menu to switch the current runtime view (use Node 16 for scripted APIs and Chrome 97 for scripted browser monitors). 
+4. Click **Validate** to check that your monitors function in the new runtime. Make any script script modifications if needed. 
+5. Save.
+
+## Switch to the legacy runtime from the UI [#runtime-UI]
+
+You have the option to toggle between the legacy environment and the new runtime environment when you create a new monitor. While the legacy runtime and its features still have backwards support, we recommend that you convert to the new runtime environment as soon as possible. 
+
+1. Go to [one.newrelic.com](https://one.newrelic.com/nr1-core), then click **Synthetics**.
+2. Click **Create monitor**, then select the public monitor type you want to create.
+3. Use the **Runtime** dropdown menu to switch between the legacy and new runtime environment. 
+4. Create your monitor. 
+
+## Deprecated features [#deprecated]
+
+We’re introducing new language that changes the script syntax in your scripted browser monitors. We recommend that you update your scripts to account for language changes and prevent potential broken monitors in the future. 
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "100px" }}>
+        Deprecated syntax
+      </th>
+      <th>
+        New syntax
+      </th>
+      <th>
+        Why?
+      </th>
+    </tr>
+  </thead>
+
+    <tbody>
+      <tr>
+        <td>
+          `$browser`
+        </td>
+        <td>
+          `$webDriver`
+        </td>
+        <td>Gives your scripts full compatibility with Selenium WebDriver 4.1 syntax.
+        </td>
+      </tr>
+      
+      <tr>
+        <td>
+          `$driver`
+        </td> 
+        <td>
+          `$Selenium`
+        </td>
+        <td>Gives your scripts full compatibility with Selenium WebDriver 4.1 syntax.
```

Comment:
We could mention that $driver provides backwards compatibility with Selenium WebDriver 3.6 while $selenium allows you to use Selenium WebDriver 4.1 syntax.